### PR TITLE
Update the emailconfig to Laravel 11 from Laravel 5

### PR DESCRIPTION
**Pull request by ysbrandB**
This hopefully fixes the ehlo error we are getting from google.

### CODE REVIEW

File: config/mail.php [hunk 1]
```diff
@@ -4,45 +4,91 @@
...
+    |
+    */
 
     'markdown' => [
-        'theme' => 'default',
+        'theme' => env('MAIL_MARKDOWN_THEME', 'default'),
+
         'paths' => [
             resource_path('views/vendor/mail'),
         ],
```
Comment (Menkveld-24) [inline config/mail.php:94]: Where does this come from?
I dont see it in the laravel 11 mail config?

https://github.com/laravel/laravel/blob/11.x/config/mail.php

File: config/mail.php [hunk 2]
```diff
@@ -4,45 +4,91 @@
...
+    | their respective settings. Several examples have been configured for
+    | you and you are free to add your own as your application requires.
+    |
+    | Laravel supports a variety of mail "transport" drivers that can be used
+    | when delivering an email. You may specify which one you're using for
+    | your mailers below. You may also add additional mailers if needed.
+    |
+    | Supported: "smtp", "sendmail", "mailgun", "ses", "ses-v2",
+    |            "postmark", "resend", "log", "array",
+    |            "failover", "roundrobin"
```
Comment (Menkveld-24) [inline config/mail.php:34]: As you've just copied array, smtp and log. Perhaps remove the others or add this as a reference: https://github.com/laravel/laravel/blob/11.x/config/mail.php

File: config/mail.php (hunk 1 above)
Comment (ysbrandB) [reply config/mail.php:94]: This came from the config in our laravel vendor folder

**Review by Menkveld-24 — APPROVED** [review]
Muy bien

**Pull request merged**